ui/webui/Makefile.am

```diff
@@ -59,22 +60,34 @@ install-data-hook: $(WEBPACK_TEST)
 
 EXTRA_DIST = dist src
 
-# checkout Cockpit's PF/React/build library; again this has no API stability guarantee, so check out a stable tag
-$(LIB_TEST):
-	git clone -b 260 --depth=1 https://github.com/cockpit-project/cockpit.git tmp/cockpit
-	mv tmp/cockpit/pkg/lib src/
-	rm -rf tmp/cockpit
+# checkout common files from Cockpit repository required to build this project;
+# this has no API stability guarantee, so check out a stable tag when you start
+# a new project, use the latest release, and update it from time to time
+COCKPIT_REPO_FILES = \
+	pkg/lib \
+	test/common \
+	tools/git-utils.sh \
+	tools/make-bots \
+	$(NULL)
+
+COCKPIT_REPO_URL = https://github.com/cockpit-project/cockpit.git
+COCKPIT_REPO_COMMIT = 6073b2703acd68e216bd9dbc116c30d2d7a9701c # 288 + esbuild-common
```

Comment:
In Cockpit projects we incidentally update it, usually when we fix something so that's not something to go by. I believe there is an issue to update automatically use a GH action. I think it might make sense to do it every 2/4 weeks based on the last Cockpit release.

So the action would fetch the last Cockpit release commit and create a PR.